governance/votes/Executive Vote - November 12, 2021.md

```diff
@@ -0,0 +1,84 @@
+
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- Changes to the GUNIV3DAIUSDC-A [Maximum Debt Ceiling (line)](https://makerdao.world/en/learn/governance/module-dciam), [Liquidation Ratio](https://makerdao.world/en/learn/governance/param-liquidation-ratio), and [Stability Fee](https://makerdao.world/en/learn/governance/param-stability-fee), as detailed below
+- [Maximum Debt Ceiling (line)](https://makerdao.world/en/learn/governance/module-dciam) parameter for WSTETH-A will be increased to 50 million DAI
+- Changes to the DIRECT-AAVEV2-DAI [Maximum Debt Ceiling (line)](https://makerdao.world/en/learn/governance/module-dciam), [Target Available Debt (gap)](https://makerdao.world/en/learn/governance/module-dciam), and Target Borrow Rate, as detailed below
+- 218,059.10 DAI will be returned to the Surplus Buffer from PauseProxy
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+$OfficeHours? This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. This is to ensure that at least some auction keepers are available to react to unforeseen issues with their bots in the event of early liquidations on new collateral types.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### GUNIV3DAIUSDC-A Parameter Adjustments
+
+As per [this poll](https://vote.makerdao.com/polling/QmemHGSM?network=mainnet), the following GUNIV3DAIUSDC-A parameter adjustments will occur, if this executive passes:
+
+* Increase the [Maximum Debt Ceiling (line)](https://makerdao.world/en/learn/governance/module-dciam) from 50 million DAI to 500 million DAI
```

Comment:
So for these we usually use the form: 'The x will be increased/decreased from...'
Basically future tense saying 'this will happen if this passes' rather than present tense. This might need addressing throughout.